config/initializers/climate_watch_engine.rb

```diff
@@ -0,0 +1,15 @@
+ClimateWatchEngine.s3_bucket_name = Rails.application.secrets.s3_bucket_name
```

Comment:
Aren't these only required when importing this datasets? do we need them to be always initialised when Rails starts?

Reply:
It's in the initializer, because this is the way to pass parameters to the engine. The importer is now part of the engine and the path to the file needs to be configurable; maybe it could be done using a service decorator in the host application rather than initializer, but that would make engine integration more cumbersome I think.

Reply:
OK! =) It seems to make sense and I trust it does =)